src/applications/vaos/redux/selectors.js

```diff
@@ -72,4 +72,4 @@ export const selectFeatureFilter36Vats = state =>
   toggleValues(state).vaOnlineFilter36Vats;
 
 export const selectFeatureVaosV2Next = state =>
-  toggleValues(state).vaOnlineSchedulingVaosV2Next;
+  toggleValues(state).vaOnlineSchedulingVaosv2next;
```

Comment:
Why the name change? The flag does not seem to work locally unless we also replace the name on src/applications/vaos/services/mocks/index.js and src/platform/utilities/feature-toggles/featureFlagNames.js  in order for the flag to work on our local development. Luckily, this name change does not seem to impact vet-api toggle switch in staging environment, just our local environment. I suggest keeping the old name `vaOnlineSchedulingVaosV2Next`. 
```suggestion
  toggleValues(state).vaOnlineSchedulingVaosV2Next;
```

Reply:
@jenniemc  - When using the review instance the vaOnlineSchedulingVaosV2Next was undefined. I don't know why but it seems to work with the change. I'm looking further into this.

Reply:
Seems to be working now.